test/Microsoft.AspNet.Mvc.TagHelpers.Test/OptionTagHelperTest.cs

```diff
@@ -15,79 +14,141 @@ public class OptionTagHelperTest
     {
         // Original content, selected attribute, value attribute, selected values (to place in FormContext.FormData)
         // and expected output (concatenation of TagHelperOutput generations).
-        public static TheoryData<string, string, string, ICollection<string>, string> GeneratesExpectedDataSet
+        public static TheoryData<string, string, string, ICollection<string>, TagHelperOutput> GeneratesExpectedDataSet
         {
             get
             {
-                return new TheoryData<string, string, string, ICollection<string>, string>
+                return new TheoryData<string, string, string, ICollection<string>, TagHelperOutput>
                 {
-                    { null, null, null, null,
-                        "<not-option label=\"my-label\"></not-option>" },
-                    { null, string.Empty, "value", null,
-                        "<not-option label=\"my-label\" value=\"value\" selected=\"\"></not-option>" },
-                    { null, "selected", "value", null,
-                        "<not-option label=\"my-label\" value=\"value\" selected=\"selected\"></not-option>" },
-                    { null, null, "value", new string[0],
-                        "<not-option label=\"my-label\" value=\"value\"></not-option>" },
+                    // original content, selected, value, selected values,
+                    // expected tag helper output - attributes, content
+                    {
+                        null, null, null, null,
+                        GetTagHelperOutput(
+                            "not-option", new Dictionary<string, string> { { "label", "my-label" } }, "")
+                    },
+                    {
+                        null, string.Empty, "value", null,
+                        GetTagHelperOutput(
+                            "not-option",
+                            new Dictionary<string, string> {
+                                { "label", "my-label" }, { "value", "value" }, { "selected", "" } },
+                            "")
+                    },
```

Comment:
nit because this is test code but the code is harder to read than it needs to be: (not Java so) no curly braces at the end of lines.  of less importance, should also avoid orphans arguments.  that is,
``` c#
{
  null, string.Empty, "value", null,
  GetTagHelperOutput(
    "not-option",
    new Dictionary<string, string>
    {
      { "label", "my-label" }, { "value", "value" }, { "selected", "" },
    },
  "")
},
```
or
``` c#
{
  null,
  string.Empty,
  "value",
  null,
  GetTagHelperOutput(
    "not-option",
    new Dictionary<string, string>
    {
      { "label", "my-label" }, { "value", "value" }, { "selected", "" },
    },
  "")
},
```